src/main/java/seedu/duke/project/Project.java

```diff
@@ -0,0 +1,70 @@
+package seedu.duke.project;
+
+import seedu.duke.Member;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Project {
+
+    public static ProjectBacklog backlog;
+    public static ProjectMember member;
```

Comment:
I think it would be better if you change this to plural form since it contains a list of members. Likewise for the class name.